src/components/Header/Header.tsx

```diff
@@ -0,0 +1,59 @@
+import { Box, Button, Text, Avatar } from "@chakra-ui/react";
+import { DragHandleIcon, BellIcon } from "@chakra-ui/icons";
+import "./Input.scss";
+
+const Header = () => {
+  return (
+    <Box ml="18%" mr="2%" display="flex" justifyContent="space-between">
+      <Box>
+        <Box
+          fontSize={30}
+          color="white"
+          fontWeight="bold"
+          fontFamily="'Barlow', sans-serif"
+        >
+          List
+        </Box>
+        <Box>
+          <Text color="#5B5A99" fontSize={13} fontFamily="'Barlow', sans-serif">
```

Comment:
Read about chacra themes, and way they store global color variables, if you want you can try that here.